openhivenpy/types/embed.py

```diff
@@ -6,7 +6,7 @@
 
 logger = logging.getLogger(__name__)
 
-class Embed():
+class Embed(): #wait hiven has embeds?
```

Comment:
Yes it has but currently only a low-level form

Reply:
Intriguing